ansible/roles/stack-sunbird/templates/analytics_api_service.conf

```diff
@@ -156,6 +156,10 @@ elasticsearch.index.compositesearch.name="compositesearch"
 elasticsearch.index.dialcodemetrics.name="dialcodemetrics"
 metrics.dialcodemetrics.request.limit=1000
 dataexhaust.authorization_check=true
+user.profile.url="{{ user_profile_read_url }}"
+org.search.url="{{ org_search_url }}"
+standard.dataexhaust.roles=["ADMIN","ORG_ADMIN","REPORT_ADMIN"]
+ondemand.dataexhaust.roles=["ADMIN","ORG_ADMIN","REPORT_ADMIN","COURSE_ADMIN"]
```

Comment:
Here the roles should be ORG_ADMIN, REPORT_ADMIN, CONTENT_CREATOR, COURSE_MENTOR

Reply:
Done - updated